aixplain/factories/benchmark_factory.py

```diff
@@ -237,3 +238,24 @@ def list_normalization_options(cls, metric: Metric, model: Model) -> List[str]:
             error_message = f"Listing Normalization Options: Error in getting Normalization Options: {e}"
             logging.error(error_message, exc_info=True)
             return []
+
+    @classmethod
+    def get_benchmark_job_scores(cls, job_id):
+        def __get_model_name(model_id):
+            model = ModelFactory.get(model_id)
+            supplier = str(model.supplier)
+            try:
+                if isinstance(supplier, Supplier):
```

Comment:
Supplier is a trick field. Check the different ways we handled in here to avoid bugs:

https://github.com/aixplain/aiXplain/blob/main/aixplain/modules/asset.py#L53C9-L61C14

Reply:
I saw a case where supplier was a dict but in form of a string, hence the extra condition. Since we do not work on raw data from BE API, all the cases handled in model factory make sure that supplier is of instance Supplier or a string in worst case. So benchmark factory does not need to again do the same.